pages/policies/[id].js

```diff
@@ -17,35 +17,45 @@
 import React from "react";
 import Link from "next/link";
 import { useRouter } from "next/router";
-import { useFetch } from "hooks/useFetch";
 import Loading from "components/Loading";
 import styles from "styles/modules/Policy.module.scss";
 import { useTheme } from "providers/theme";
 import PolicyBreadcrumbs from "components/policies/PolicyBreadcrumbs";
+import Button from "components/Button";
+import { usePolicy } from "hooks/usePolicy";
 
 const Policy = () => {
   const router = useRouter();
   const { theme } = useTheme();
 
   const { id } = router.query;
 
-  const { data, loading } = useFetch(id ? `/api/policies/${id}` : null);
+  const { policy, loading } = usePolicy(id);
+
+  const editPolicy = () => {
+    router.push(`/policies/${id}/edit`);
+  };
 
   return (
     <div className={`${styles[theme]} ${styles.pageContainer}`}>
       <PolicyBreadcrumbs />
       <div className={styles.detailsContainer}>
         <Loading loading={loading}>
-          {data ? (
+          {policy ? (
             <>
-              <div>
-                <p className={styles.policyName}>{data.name}</p>
-                <p className={styles.policyDescription}>{data.description}</p>
+              <div className={styles.detailsHeader}>
+                <div>
```

Comment:
Can we just define this div to extend to the Edit Policy button? Or whatever solution is recommended.

What's happening now is that the policyName dictates the size of the div. The subsequent policyDescription has a max-width of 90%. That's 90% of the div defined by the policyName. So a small policyName means the description can be many lines long

Reply:
Could you post a screenshot of what you're seeing? I have a small policy name with a long description, but it does not look like the constraint is to the policy name like you're saying so I just want to make sure I'm doing you justice

![image](https://user-images.githubusercontent.com/25139211/112843857-aa1da480-9068-11eb-8287-7b6a1eaaca70.png)

Reply:
@aalsabag check out the latest changes I pushed, they might fix the issue you were seeing? I was definitely not handling super long policy names, and your comment reminded me so thanks! Hopefully all should be good now.